_config/config.yml

```diff
@@ -7,7 +7,9 @@ After:
 Director:
   rules:
    'timednotice//$Action/$ID/$OtherID': 'TimedNoticeController'
-
+Page:
+  extensions:
+    - TimedNoticedPageExtension
```

Comment:
Rather than enforcing the use of this extension, can this instead be included as part of the installation/configuration section of the readme.md?